### DISCUSSION TITLE
TeamTalk javascript bot

### DISCUSSION THREAD

**Discussion by ereziel** [BearWare/TeamTalk5#1537] [Ideas]
Hello,
I suggest to have a module, a guide and a documentation to create bots for TeamTalk in javascript (node.js).

**Comment by beqabeqa473** [comment]
Hello.

First of all you should write a module to communicate with teamtalk
through the tcp protocol using net.sockets for example.

On 10/9/22, Canadarm2 ***@***.***> wrote:
> Hello,
> I suggest to have a module, a guide and a documentation to create bots for
> TeamTalk in javascript (node.js).
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/BearWare/TeamTalk5/discussions/1537
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili

**Reply by ereziel** [reply]
> Hello. First of all you should write a module to communicate with teamtalk through the tcp protocol using net.sockets for example.
> […](#)
> On 10/9/22, Canadarm2 ***@***.***> wrote: Hello, I suggest to have a module, a guide and a documentation to create bots for TeamTalk in javascript (node.js). -- Reply to this email directly or view it on GitHub: #1537 You are receiving this because you are subscribed to this thread. Message ID: ***@***.***>
> -- with best regards Beqa Gozalishvili Tell: +995593454005 Email: ***@***.*** Web: https://gozaltech.org Skype: beqabeqa473 Telegram: https://t.me/gozaltech facebook: https://facebook.com/gozaltech twitter: https://twitter.com/beqabeqa473 Instagram: https://instagram.com/beqa.gozalishvili

Hello,
But I don't know how to write a module...
